TGEngine/pipeline/Swapchain.hpp

```diff
@@ -14,18 +14,23 @@
 #include "../ui/UIEntity.hpp"
 
 extern VkSwapchainKHR swapchain;
-extern std::vector<VkImage> swapchain_images;
-extern VkSurfaceCapabilitiesKHR surface_capabilities;
-extern VkImage color_image;
-extern VkImageView color_image_view;
-extern VkDeviceMemory color_image_memory;
+extern VkImage* swapchainImages;
+/*
+ * Contains the supported image usage flags
+ * The min and max image count as well as
+ * min and max extend
```

Comment:
extend? do you mean extension? who knows

Reply:
I meant extent ...